grant/projects/evolutionland.md

```diff
@@ -0,0 +1,118 @@
+
+Evolution Land integrates various gameplays, including management, collection, education, gene, battle, auction, and game, which will continuously be evolving.
+
+**One Dapp One Community Metaverse**
+10+ projects have announced plans to build communities in Evolution Land to present project information and give back rewards to the community fans.
+
+### **Why is this project is good for the Crab ecosystem:**
+
+1. Evolution Land is the first build GameFi project on Crab Smart Chain. Evolution Land has been built on four network - Ethereum, Tron, HECO, and Polygon. Each of them have at least 200,000+ $RING TVL. It will definately bring more TVL on Crab Smart Chain.
+2. Evolution Land have 1000+ DAU from HECO, Ethereum, Tron, and Polygon. And will build on BSC and Moonriver in the future. It could introduce more users and assets from other chains.
+3. Multi-chain and cross-chain is an important gameplay on Evolution Land. This point is similar to the Darwinia Bridge.
+
+### **How you will integrate this project into Crab:**
+
+Evolution Land will take the advantage of EVM module on Crab Smart Chain. We will deploy our contract(ERC20, ERC721, ERC1155) on CSC. We alreay deployed our alpha version for testing. Once our beta is released we will enable the community and Darwinia to start minting and playing.
+
+### **Team**
+
+- Tiny Liu ( Project Manager of Evolution Land )
+- Shell Wan ( Front-end Engineer )
+- Perror Qin ( Backend Engineer )
+- Echo Hu ( Solidity Engineer)
+- Xinyue Wang ( UI/UX Designer )
+
+### **Legal Structure**
+
+Currently Evolution Land has no defined legal structure other then a Dapp created by Evolution Land team.
+
+## **Team Code Repos**
+
+- [https://github.com/evolutionlandorg](https://github.com/evolutionlandorg) (Evolution Land)
+
+## **Collaborative Grant Proposal and Current Development**
+
+### **Proposal**
+
+1. Build and deploy Evolution Land on Crab Network: a **Metaverse+Gamefi+cross-chain game**. 
+2. Start marketing events on Crab continent - Columbus.
+
+Evolution Land will provide the following to Darwinia and Crab community:
+
+- Latest version including PVE, equipment, and profession gameplay
+- Continously update on Columbus continent
+- Farm pool for users to stake $xRING
+- 70% of profit will return to users
+
+## Evolution Land **Beta Version:**
+
+The beta version of Evolution Land is what is applying for CRAB grant program, and the delivery scope has:
+
+- Basic mining gameplay including drills and apostles mining functions
+- Apostle Adventure - PVE gameplay
+- Furnance System - First NFT + FT Gamefi gameplay
+- Buy, sell, destroy and send functions
+- Apostle breeding system
+- Profession system
+
+## **Future Development Roadmap**
+
+The following milestones are not included in the collaboration grant assessment and is only here to communicate Evolution Land team's vision and direction.
+
+### Evolution Land **1.0**
```

Comment:
Why is 1.0 scope the same as beta version?

Reply:
beta version is only for test, 1.0 scope would be the formal one